serve/package.go

```diff
@@ -111,7 +113,7 @@ func (s *PluginServe) build(pluginDirectory, goos, goarch, distPath, pluginVersi
 	if err != nil {
 		return nil, err
 	}
-	ldFlags := fmt.Sprintf("-s -w -X %s/plugin.Version=%s", importPath, pluginVersion)
+	ldFlags := fmt.Sprintf("-s -w -X %s/plugin.Version=%s -X %s/resources/plugin.Version=%s", importPath, pluginVersion, importPath, pluginVersion)
```

Comment:
```suggestion
	ldFlags := fmt.Sprintf("-s -w -X %[1]s/plugin.Version=%[2]s -X %[1]s/resources/plugin.Version=%[2]s", importPath, pluginVersion)
```

(untested but should work)